### ISSUE TITLE
Automatic reload

### ISSUE THREAD

**Issue by mvasin** [jakubboucek/schedule-reception-doctors#2]
Currently is external data loaded only when user manually reload page. It can be refresh automatically.

I think this one is mandatory, we need to create an interval parameter and fetch the JSON according to it.

**Comment by jakubboucek** [comment]
Depended on #4.

**Comment by jakubboucek** [comment]
What browser you will use for open app? I am thinking about build simple Chrome extension.

**Benefits:** 
1. It can safe restart app when it cause master failure
2. it can close and open new tab (rotate process thread - it can help prevent memory leaks on long running app)

Related to #3 too.

**Comment by mvasin** [comment]
No, there will be a random rudimentary weird browser that you can find in not-so-fresh smart TVs. An exact model is not known.

**Comment by jakubboucek** [comment]
[Noooooo!](http://www.nooooooooooooooo.com/) Please can you just now testing app on some of target devices? Because app was from previous developer builded on modern HTML5 features (some [ES6 features](http://es6-features.org/), [Flexbox](http://caniuse.com/#search=flexbox)) and I continued on it.

**Comment by mvasin** [comment]
Don't worry, we can stick to current modern technologies, but we just can't expect a conventional PC with Chrome on it.

**Comment by mvasin** [comment]
I tried it on my two years old Toshiba. Here's what I saw: https://drive.google.com/file/d/0B4D93ILRdf51RkkxRkhtSEVraTg/view

The pages don't split, every clinic is on its own single page, so there are two pages for two clinics.
It has scroll bars both horizontal and vertical.

If making our code compatible with at least this TV requires tremendous efforts, I can _try_ to persuade the customer to use some external console for displaying web content. But if we can make it play on my TV, I'd be in a much more confident position, I could say: "at least here's the TV that plays it fine, and we cannot support every outdated device".

**Comment by jakubboucek** [comment]
Pls move this case to unique issue. I will next hour on PC a I will try to check it.

**Comment by 7flash** [comment]
Automatic reload is not a with current code state.

**Comment by mvasin** [comment]
@7flash Yes, and we need to fix this. That's what the issue is all about.
